src/main/java/Display.java

```diff
@@ -0,0 +1,48 @@
+import java.util.ArrayList;
+import java.util.List;
+
+public class Display {
+    public void printGreeting() {
+        System.out.println("Welcome to TicTacToe!");
+    }
+
+    public void promptPlayer(String playerMark) {
+        System.out.println(String.format("Make a move: %s", playerMark));
+    }
+
+    private List currentGridCells(Board board, int size) {
+        List currentGridAsArray = new ArrayList<>();
+        Board boardClass = board;
+        int gridSize = size;
+        int count = 1;
+
+        while (count <= gridSize) {
+            String cell = boardClass.getCellAtPosition(count);
```

Comment:
Is the purpose of this method just to convert your list of ints into a list of strings to be printed below? I think you could simplify this by just going through the board positions when you're printing it. Java is going to coerce those ints into strings for you.

Reply:
✅ changed to `for` loop PR #6